app/services/yelp_services.rb

```diff
@@ -0,0 +1,17 @@
+class YelpService
+  attr_reader :restaurant
+
+  def initialize(latitude, longitude, radius, price)
+    conn = Faraday.new("https://api.yelp.com/v3/businesses/search") do |f|
```

Comment:
Could this Faraday.new be pulled out into private? Probably unnecessary for our app at this point but maybe for future iterations?

Reply:
I might have read this wrong too and it is already broken up? @holmesm8 @caachz